src/odemis/model/_metadata.py

```diff
@@ -89,6 +89,7 @@
 # position of the stage (in m or rad) for each axis in the chamber (raw hardware values)
 MD_STAGE_POSITION_RAW = "Stage position raw"  # dict of str -> float,
 MD_SAMPLE_PRE_TILT = "pre-tilt"  # (rad) pre-tilt of the sample stage / shuttle (tilt)
+MD_MEDIAN_FILTER = "median-filter" # median filter applied to detector data (req for cryo)
```

Comment:
Let's drop this. If you go with the filter inside the driver, then introduce MD_DATA_FILTER = "Data filter"